packages/uploadthing/tsconfig.json

```diff
@@ -14,7 +14,8 @@
     "noUnusedParameters": false,
     "preserveWatchOutput": true,
     "skipLibCheck": true,
-    "strict": true
+    "strict": true,
+    "module": "ES2020"
```

Comment:
I also set module to "ES2020" so that TypeScript won't complain about using import.meta. Is it fine or may cause some issues with? I mean current existing examples are building fine, but maybe there are edge cases I'm not aware about